src/ObjectUtils/SymbolsFile.cpp

```diff
@@ -22,12 +22,12 @@ ErrorMessageOr<std::unique_ptr<SymbolsFile>> CreateSymbolsFile(
     const std::filesystem::path& file_path, const ObjectFileInfo& object_file_info) {
   ORBIT_SCOPE_FUNCTION;
   std::string error_message{
-      absl::StrFormat("Unable to create symbols file from \"%s\":", file_path.string())};
+      absl::StrFormat("Unable to create symbols file from \"%s\": ", file_path.string())};
 
   OUTCOME_TRY(auto file_exists, orbit_base::FileOrDirectoryExists(file_path));
 
   if (!file_exists) {
-    error_message.append("\n* File does not exist.");
```

Comment:
The purpose of these "\n*" is to guarantee the readability of the error message showing in the SymbolErrorDialog.
So it might be better if we keep them.

Reply:
I know, but:
- In this `if` the error message here is returned right away, so it looked like
```
Unable to create symbols file from "/path/to/file":
* File does not exist.
```
when it can just look like
```
Unable to create symbols file from "/path/to/file": File does not exist.
```
- You have to consider that the caller of this function could already produce error messages with bullet points. When that happens, this is not indented properly, as it just merges into the bullet points of the caller, and that hurts readability just as much. Keeping track of the nesting levels is hard, and it is impossible if the function is called in different places. So I had to simplify the situation somewhere and this was the place where it hurt the least.